worker-pools.yml

```diff
@@ -1988,6 +1988,46 @@ pools:
           guestAccelerators:
             - acceleratorCount: 4
               acceleratorType: nvidia-tesla-v100
+  - pool_id: '{pool-group}/b-linux-v100-gpu-d2g'
+    description: Worker for machine learning and other high GPU tasks
+    owner: release+tc-workers@mozilla.com
+    variants:
+      - pool-group: translations-1
+    email_on_error: true
+    provider_id:
+      by-chain-of-trust:
+        trusted: fxci-level3-gcp
+        default: fxci-level1-gcp
+    config:
+      lifecycle:
+        # low inactivity timeout because these workers are very expensive
+        queueInactivityTimeout: 1800
+      worker-config:
+        genericWorker:
+          config:
+            # 2592900s is 30 days plus 900 seconds to account for https://github.com/taskcluster/taskcluster/issues/7423
+            maxTaskRunTime: 2592900
+            enableInteractive: true
+            d2gConfig:
```

Comment:
Please set allowPrivileged and allowHostSharedMemory to false everywhere.

Reply:
Might be time to have a config section for d2g workers in worker-defaults so we don't have to repeat this stuff all over the place.

Reply:
Good call on both fronts; will do.

Reply:
I'm doing this in https://github.com/mozilla-releng/fxci-config/pull/293; I'll rebase on top of it afterwards.